src/main/java/uk/gov/hmcts/reform/civil/enums/GeneralAppClaimType.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.civil.enums;
```

Comment:
where is this class being used? I see its usage only as part of tests... we should not be introducing models only for test purpose.

Reply:
we can use this later when we refactor code for spec and unspec claim.,

Reply:
We can introduce it later too when we need it.

Reply:
I have removed safely